client/src/components/PaperSearchBar/PaperSearchBar-redux.tsx

```diff
@@ -1,36 +1,22 @@
-import axios from 'axios';
 import React from 'react';
-import { constructPaperFromResponse, PaperResponse } from '../../dtos';
 import { useAppDispatch, useAppSelector } from '../../hooks/reduxHooks';
 import { setActiveReview } from '../../slices/activeReviewSlice';
-import { updateReadingList } from '../../slices/readingListSlice';
 import { blankNotes, blankReview } from '../../templates';
 import { Paper, Review } from '../../types';
+import { useUpdateReadingListFunc } from '../ReadingList/hooks';
 import PaperSearchBarContainer from './PaperSearchBar-container';
 
 export default function PaperSearchBarRedux(): JSX.Element {
   const dispatch = useAppDispatch();
   const readingList = useAppSelector((state) => state.readingList);
-
+  const { updateReadingListFunc } = useUpdateReadingListFunc();
```

Comment:
Also fine as-is, but I think the `func` suffix is clunky. `useUpdateReadingList` is descriptive enough IMO